spec/services/templates_service_spec.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'fileutils'
+
+describe TemplatesService do
```

Comment:
Add `, type: :background_job`.

Reply:
This is a service, not a background job

Reply:
Sure, but this makes the delayed jobs (which you do trigger inside the tests) to run immediately. Or am I wrong?

Reply:
If I'm correct, then maybe you wouldn't need to call delayed jobs twice at _line 17_?

Reply:
config.around(:each, type: :background_job) do |example|
    run_background_jobs_immediately do
      example.run
    end
  end

It just wraps test, jobs than are executed in the end